src/screens.ts

```diff
@@ -1,12 +1,12 @@
 // Dependencies
-const chalk = require('chalk');
-const Table = require('cli-table');
-const utils = require('./utils.js');
+import chalk from 'chalk';
+import Table from 'cli-table';
+import utils, { OrganizedArguments, Settings, CommandSpec } from './utils';
 
-// Helpful utility functions
-module.exports = function Constructor(currentSettings) {
+// Default export
+export function screens(currentSettings: Settings) {
```

Comment:
This is being exported as the default later on.

```suggestion
function screens(currentSettings: Settings) {
```